src/components/search/Suggestions/Suggestions.tsx

```diff
@@ -121,12 +121,12 @@ const Suggestions = forwardRef<HTMLDivElement, SuggestionsProps>(
         )}
 
         {SUGGESTED_PRODUCTS.length > 0 && (
-          <div className="suggestions__section">
-            <p className="suggestions__title">Suggested Products</p>
+          <div data-fs-search-suggestion-section>
+            <p data-fs-search-suggestion-title="small">Suggested Products</p>
             <UIList>
               {SUGGESTED_PRODUCTS.slice(0, MAX_SUGGESTED_PRODUCTS).map(
                 (product, index) => (
-                  <li key={index} className="suggestions__item">
+                  <li key={index} data-fs-search-suggestion-item>
```

Comment:
Same suggestion about not using the index as a key here.

Reply:
[Done](https://github.com/vtex-sites/gatsby.store/pull/7/commits/544924fcdf912f49199265a308e06ccc8fe59c36)!